blog-posts/1-announcing-swarmion/announcing-swarmion.md

```diff
@@ -0,0 +1,145 @@
+---
+published: true
+title: 'Introducing Swarmion 🐝, a framework for shipping type-safe Serverless microservices at scale'
+cover_image: TODO
+description: TODO
+tags: serverless, typescript, monorepo, microservices
+series:
+canonical_url:
+---
+
+The serverless computing paradigm introduced a lots of opportunities to create more scalable and cost-efficient software, but it also came with its lot of challenges, some of which we harshly experienced.
```

Comment:
Retour de @Sc0ra:
introduced a lots -> a lot
came with its lot of challenges -> came with its own challenges